relayertest/legacy.go

```diff
@@ -0,0 +1,96 @@
+package relayertest
+
+import (
+	"bytes"
+	"fmt"
+	"runtime"
+	"strings"
+	"sync"
+
+	"github.com/strangelove-ventures/ibc-test-framework/ibc"
+)
+
+// GetLegacyTestCase looks up the given testCase and returns a function suitable for the existing cmd structure.
+func GetLegacyTestCase(testCase string) (func(string, ibc.ChainFactory, ibc.RelayerImplementation) error, error) {
```

Comment:
There is probably a better name for this.

This wraps the `TestRelayer_*` functions in the function signature that the existing commands expect. If we take the pure `go test` path, we won't need this file.

Reply:
I think pure go test will be better, as long as the matrix file can support the current available chain configuration customization